common/sphinx.c

```diff
@@ -518,11 +518,6 @@ static bool sphinx_write_frame(u8 *dest, const struct sphinx_hop *hop)
 	size_t padding_size;
 	int pos = 0;
 
-#if !EXPERIMENTAL_FEATURES
-	if (hop->type != SPHINX_V0_PAYLOAD)
-		return false;
-#endif
-
```

Comment:
Changelog should actually announce TLV!  (Changelog-Added: We handle modern TLV-style payloads).  This is important because we never announce EXPERIMENTAL features.

Reply:
I was hoping you'd add a changelog entry to #3167, since you did all the heavy lifting there, but you're right, that we don't note experimental features in the changelog.

I'd propose the following ling:

```
Changelog-Added: Plugins may now handle modern TLV-style payloads via the `htlc_accepted` hook
```

That's a less broad claim than full TLV support for sending and receiving without a plugin, which as you noted is still experimental.